core/state/database.go

```
@@ -802,6 +803,7 @@ func (tds *TrieDbState) updateTrieRoots(forward bool) ([]common.Hash, error) {
 			}
 
 			tds.t.DeleteSubtree(addrHash[:])
+			_ = PutTombstoneForDeletedAccount(tds.db, addrHash[:])
```

Ideally, this error should not be swallowed (can do in the next PR)